src/styles/global.css

```
@@ -0,0 +1,50 @@
+button { cursor: pointer; font-family: inherit; }
+input, select { font-family: inherit; }
+a { color: var(--accent); text-decoration: none; }
+
+::-webkit-scrollbar { width: 5px; height: 5px; }
+::-webkit-scrollbar-track { background: var(--bg); }
+::-webkit-scrollbar-thumb { background: var(--border); border-radius: 3px; }
+
+@keyframes spin {
+  to { transform: rotate(360deg); }
+}
+@keyframes fadeUp {
```

Stylelint (17.12.0)

[error] Expected keyframe name "fadeUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)